lib/lightning/impact_tracking/client.ex

```diff
@@ -6,11 +6,24 @@ defmodule Lightning.ImpactTracking.Client do
   """
   use Tesla, only: [:post], docs: false
 
+  alias Lightning.ImpactTracking.ResponseProcessor
+
   def submit_metrics(metrics, host) do
-    build_client(host) |> post("/api/metrics", metrics)
+    result =
+      build_client(host)
+      |> post("/api/metrics", metrics)
+      |> elem(1)
+      |> ResponseProcessor.successful?()
+      |> indicate_success
+
+    {result, metrics}
```

Comment:
if we don't intend to change metrics and only submitting would be fair, we could return only the result (without the result variable)